lib/fizzy/limits.hpp

```diff
@@ -0,0 +1,9 @@
+namespace fizzy
+{
+namespace
+{
+constexpr unsigned page_size = 65536;
+// Set hard limit of 256MB of memory.
+constexpr unsigned memory_pages_limit = (256 * 1024 * 1024ULL) / page_size;
```

Comment:
@gumb0 @chfast we could also camelcase these in this PR if you want.

Reply:
I would like to, I think it's useful to see that a variable is a global const without jumping to definition

Reply:
@gumb0 @chfast how about this one then?